lib/providers/password.js

```diff
@@ -16,31 +16,46 @@ var errors = require('../errors');
 function PasswordProvider (client, options) {
     // Jar for storing the session cookie
     this.jar = request.jar();
+    // storage of the CSRF Token as given in the response from the server
+    this.csrfToken = null;
+    this.csrfTokenLocation = 'x-csrf-token';
```

Comment:
I felt that something that didn't have a spec deserved a place to change this header once and have all places update. I don't like hardcoding things like this.